test/test_source_comment_associator.rb

```diff
@@ -27,8 +33,6 @@ def associate_locations(code)
 
   def test_associate
     ast, associations = associate(<<-END)
-#!/usr/bin/env ruby
```

Comment:
You shouldn't remove these. They're testing the behavior of the comment associator, and this code is not ever run.